engine/src/w32-clipboard.h

```diff
@@ -61,6 +61,11 @@ class MCWin32RawClipboardItemRep :
 	MCWin32RawClipboardItemRep(MCWin32RawClipboardItem* p_parent, const FORMATETC& p_format);
 	MCWin32RawClipboardItemRep(MCWin32RawClipboardItem* p_parent, MCStringRef p_type, MCDataRef p_bytes);
 	~MCWin32RawClipboardItemRep();
+
+
+	virtual MCDataRef CopyGlobalData(STGMEDIUM *p_medium) const;
```

Comment:
These don't have to be virtual - they are private helper methods of the MCWin32RawClipboardItemRep class - which isn't overriden.